_pkgdown.yml

```
@@ -41,7 +41,7 @@ reference:
       - resolve
       - resolve_delayed
   - title: Data extract modules
-    desc: teal modules to generate transform expressions
+    desc: `teal` modules to generate transform expressions
```

```suggestion
    desc: teal modules to generate transform expressions
```

pkgdown is complaining when we start the title with "`"

---

we could put the entire sentence in doub quote

 "`teal` modules to generate transform expressions"

---

It's fixed now. I'm taking a look at the CI errors :-)